lua/trunk.lua

```diff
@@ -265,13 +320,27 @@ local function start()
 					return
 				end
 
-				local bufname = vim.fs.basename(vim.api.nvim_buf_get_name(0))
```

Comment:
has been unused for a while - for some reason trunk check didn't flag this

Reply:
Possibly stylua needs some more configuration